qiskit/pulse/commands/sample_pulse.py

```diff
@@ -88,13 +88,12 @@ def __eq__(self, other: 'SamplePulse'):
         Returns:
             bool: are self and other equal
         """
-        if super().__eq__(other) and \
-                (self._samples == other._samples).all():
+        if super().__eq__(other) and (self.samples == other.samples).all():
```

Comment:
this is where `np.isclose(samples, other.samples, epsilon)` would probably be good. is okay for now I suppose

Reply:
Yes, let's consider elsewhere.